web-animations-1/Overview.bs

```diff
@@ -3974,6 +3979,7 @@ The <code>AnimationTimeline</code> interface {#the-animationtimeline-interface}
 [Exposed=Window]
 interface AnimationTimeline {
     readonly attribute double? currentTime;
+    readonly attribute double? duration;
```

Comment:
I guess this needs to be `unrestricted double` in order to represent infinity?

Reply:
I switched this to having the document timeline (i.e. non-finite timelines) have a null duration leading to a 0-time but let me know if we should switch this back.